src/components/header/Header.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import CssBaseline from '@material-ui/core/CssBaseline';
```

Comment:
Could you try this approach : import { CssBaseline, Drawer, AppBar } from '@material-ui/core';